### ISSUE TITLE
Add an API to disable TransformControls's XYZ, XY, YZ, XZ and the extra 'x', 'y' and 'z' in the visual controls.

### ISSUE THREAD

**Issue by thecowmilk** [mrdoob/three.js#27627]
### Description

When I'm attaching `TransformControls` to an object it automatically gets 13 children which are the navigation children. By default is okay but it is very frustrating when there are many objects and you are trying to move the objects using the arrows. Looks very complicated and there is no reason for them to exist. To give more context here's a screenshot:

![image](https://github.com/mrdoob/three.js/assets/43875051/0cce8d20-4ff1-431e-8504-c233504bf24e)
I don't get this complication. Okay, we can disable(sort of) them by doing this:

```
[228, 231, 234, 236, 237, 238, 239].forEach(axisMetal => {
    const objM = gizmoMetal.translate.getObjectById(axisMetal);
    objM.visible = false;
    objM.layers.disable(0);
});
```
by this is just disabling the visibility not the functionality so when there are two object near and wonder why both object moves check for an invisible working arrow and not only this but everytime a new object is added, the IDs change so good luck when you have 100+ objects.

### Solution

I think we should have the ability to disable them and by using `TransformControls` directly like this:
```
const transformControlsMetal = new TransformControls(camera, renderer.domElement);
transformControlsMetal.showXYZ = false;
```

This way we will disable the entire functionality of things we don't need.

### Alternatives

Using this will make it a partial solution but once the IDs change then the correct IDs should be updated again.
```
[228, 231, 234, 236, 237, 238, 239].forEach(axisMetal => {
    const objM = gizmoMetal.translate.getObjectById(axisMetal);
    objM.visible = false;
    objM.layers.disable(0);
});
```

**Comment by Mugen87** [comment]
Can you explain why the existing properties `showX`, `showY` and `showZ` are not sufficient?

**Comment by thecowmilk** [comment]
> Can you explain why the existing properties `showX`, `showY` and `showZ` are not sufficient?
> 
> They can be tested in the respective example by pressing the `x`, `y` and `z` keys: https://threejs.org/examples/misc_controls_transform

The reason why `showX`, `showY` and `showZ` is not sufficient is because it will also disable the main arrows:

![image](https://github.com/mrdoob/three.js/assets/43875051/7225d7bd-52e7-4c77-9ccb-701f321e5cfc)

Whilst the arrows without the line and the squares just add more complexity to the visual, controller and code.

**Comment by Mugen87** [comment]
This is not the first time devs come up with such a feature request since not everybody is happy with the gizmos, see #22045. Maybe we should try to find a solution that enables more flexibility.

The problem is the more detailed configuration options we introduce, the harder it will be to manage the controls. How about we start with `showXY`, `showYZ` and `showXZ` for toggling the  2-axis handles? This feature has been requested more than once so it's seems appropriate to implement it.

Would you be interested in making a PR? I will review and guide you through the process.

**Comment by thecowmilk** [comment]
> This is not the first time devs come up with such a feature request since not everybody is happy with the gizmos, see #22045. Maybe we should try to find a solution that enables more flexibility.
> 
> The problem is the more detailed configuration options we introduce, the harder it will be to manage the controls. How about we start with `showXY`, `showYZ` and `showXZ` for toggling the 2-axis handles? This feature has been requested more than once so it's seems appropriate to implement it.
> 
> Would you be interested in making a PR? I will review and guide you through the process.

Yeah I know that feeling and I would gladly write the PR. Receiving the guide would make the process more efficient and easy since it's the first time for me. Can we link up on Discord, maybe?

**Comment by Mugen87** [comment]
There is the following guide with some basic information about contributing: https://github.com/mrdoob/three.js/blob/dev/.github/CONTRIBUTING.md

As soon as you have opened your PR, I start with the review process.
